db/utils.js

```diff
@@ -18,17 +18,15 @@ Model.knex(knex);
 const DBUtils = {
   async createBreach(name, meta) {
     try {
-      return await Breach
-        .query()
-        .insert({ name, meta });
+      const insertedBreach = await Breach.query().insert({ name, meta });
```

Comment:
What's the reason for this change?

Reply:
Oh, I think that's left-over when I was debugging by `console.log`ging the `insertedBreach` value. I can change it back.